.travis.yml

```diff
@@ -67,3 +65,9 @@ script:
   - docker exec -w /ndmg travis ndmg_bids --sp $SP --mod $MOD --participant_label 0025864 --session_label 1 --atlas desikan --seeds 1 /input /output
   - kill %1
   - docker exec -w /ndmg travis pytest -v --disable-warnings
+  - rand=$(date +'%m-%d-%Y'-$SP-$MOD-${TRAVIS_COMMIT:0:7})
+  - heat=$(docker exec travis ls /output/sub-0025864/ses-1/qa/graphs_plotting)
+  - csv=$(docker exec travis ls /output/sub-0025864/ses-1/dwi/roi-connectomes/desikan_space-MNI152NLin6_res-2x2x2)
+  - docker exec travis aws s3 sync /output/sub-0025864/ses-1/qa/graphs_plotting s3://ndmg-data/Travis-Output/$TRAVIS_BRANCH/$rand
+  - docker exec travis aws s3 sync /output/sub-0025864/ses-1/dwi/roi-connectomes/desikan_space-MNI152NLin6_res-2x2x2 s3://ndmg-data/Travis-Output/$TRAVIS_BRANCH/$rand
```

Comment:
remember that there's a local `output` and `input` folder (outside of docker) that we created above (in the `docker run` command), so you should just be able to pull from that, rather than calling `docker exec travis`

Just a bit cleaner

Reply:
the aws functions/libraries aren't on the travis instance, so instead of installing them I just decided to have the container do it (since it's already required there)

Reply:
Ok cool, that's fine